docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -462,98 +450,81 @@ This feature is available in Enterprise Edition only.
    For example, here is how you can pull images from a third-party registry:
 
    ```shell
-    DHCTL_CLI_MIRROR_SOURCE_LOGIN="user" DHCTL_CLI_MIRROR_SOURCE_PASSWORD="password" dhctl mirror --source="corp.company.com/sys/deckhouse" --images-bundle-path /tmp/d8-images/d8.tar
+   d8 mirror pull --source="corp.company.com/sys/deckhouse" --source-login="<USER>" --source-password="<PASSWORD>" $(pwd)/d8.tar
    ```
 
-   > Note: `--license` flag acts as a shortcut for `--source-login ($DHCTL_CLI_MIRROR_SOURCE_LOGIN` and `--source-password ($DHCTL_CLI_MIRROR_SOURCE_PASSWORD)` flags for the Deckhouse registry.
+   > Note: `--license` flag acts as a shortcut for `--source-login` and `--source-password` flags for the Deckhouse registry.
    > If you specify both license and login+password pair for source registry, the latter will be used.
 
-   `dhctl mirror` supports digesting of the final set of Deckhouse images with the GOST R 34.11-2012 (Stribog) hash function (the `--gost-digest` parameter).
+   `d8 mirror pull` supports digesting of the final set of Deckhouse images with the GOST R 34.11-2012 (Streebog) hash function (the `--gost-digest` parameter).
    The checksum will be logged and written to a file with the `.tar.gostsum` extension next to the tar-archive containing the Deckhouse images.
 
-   Starting from version 1.59.0, `dhctl mirror` supports splitting the images bundle into chunks of arbitrary size instead of dumping all images into a single tar file.
-   To use this function, pass the desired chunk size in gigabytes using the `--images-bundle-chunk-size=N` flag or set the `$DHCTL_CLI_MIRROR_IMAGES_BUNDLE_CHUNK_SIZE` environment variable when pulling the bundle.
+   Also, `d8 mirror pull` supports splitting the images bundle into chunks of arbitrary size instead of dumping all images into a single tar file.
```

Comment:
```suggestion
   Also, `d8 mirror pull` supports splitting the image bundle into chunks of arbitrary size instead of dumping all the images into a single tar file.
```